gramps/plugins/importer/importxml.py

```diff
@@ -318,7 +318,7 @@ def info_text(self):
                 merge = True
                 break
         if merge:
-            txt += _("\n\nObjects that are candidates to be merged:\n")
+            txt += _("\nObjects that are candidates to be merged:\n")
```

Comment:
```suggestion
            txt += _("\nObject candidates to merge:\n")
```
Not that I know what this is…